### ISSUE TITLE
provide API

### ISSUE THREAD

**Issue by justincormack** [docker/hub-tool#112]
I think we should provide API/SDK support for these operations too. cc @chris-crone

**Comment by tianon** [comment]
Sounds like a (partial?) duplicate of #16 :eyes:

**Comment by chris-crone** [comment]
@tianon Yep it is :) We should separate out the Hub API bits and the Hub SDK bits.
